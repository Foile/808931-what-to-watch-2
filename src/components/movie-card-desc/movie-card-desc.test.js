import React from "react";
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import MovieCardDesc from "./movie-card-desc";

it(`MovieCardDesc correctly renders after relaunch`, () => {
  const movie = {id: 1, name: `Macbeth`, previewImage: ``, previewVideoLink: ``, starring: []};
  const tree = renderer
  .create(<BrowserRouter>
    <MovieCardDesc
      movie = {movie}
      match = {{params: {id: 1}}} />
  </BrowserRouter>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
