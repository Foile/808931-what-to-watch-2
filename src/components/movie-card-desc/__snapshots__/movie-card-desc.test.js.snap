// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardDesc correctly renders after relaunch 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          href="/films/1/overview"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          href="/films/1/details"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          href="/films/1/reviews"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-card__text movie-card__row"
  >
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Director
        </strong>
        <span
          className="movie-card__details-value"
        />
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          className="movie-card__details-value"
        />
      </p>
    </div>
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          className="movie-card__details-value"
        >
          undefined m
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          className="movie-card__details-value"
        />
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Released
        </strong>
        <span
          className="movie-card__details-value"
        />
      </p>
    </div>
  </div>
</div>
`;
